Frequently_used_code/ARD_overpass_predictor.ipynb

```diff
@@ -0,0 +1,864 @@
+{
```

Comment:
<p>Same as above: can we remove the commented out lines here? Anyone who wants to look at the data can always edit and add them in themselves</p><p> </p><pre class="ql-syntax" spellcheck="false">#S2B_combined
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>